Source/mpqapi.cpp

```diff
@@ -458,8 +458,6 @@ static int mpqapi_get_hash_index(int index, int hash_a, int hash_b, int locale)
 			continue;
 		if (cur_archive.sgpHashTbl[idx].hashcheck[1] != hash_b)
 			continue;
-		if (cur_archive.sgpHashTbl[idx].lcid != locale)

```

Comment:
Is this purely a warning fix?

Reply:
Unused, the easiest way to fix the related warning was to remove it.